### PR TITLE
Add isort

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ repos:
       hooks:
           - id: black
             language_version: python3.9
-    - repo: https://github.com/asottile/add-trailing-comma
```
Comment (pmac) [inline .pre-commit-config.yaml:7]: Is this functionality part of isort or something?

**Review by pmac — APPROVED** [review]
🌴

File: .pre-commit-config.yaml (hunk 1 above)
Comment (robhudson) [reply .pre-commit-config.yaml:7]: No, but I was trying to be more consistent with bedrock which doesn't have it. If it is preferred I can keep it in there and consider adding it to bedrock.

Comment (pmac) [reply .pre-commit-config.yaml:7]: I think it used to be in bedrock. Maybe black does it now? I'm not sure.

**Pull request merged**